Assets/Scenes/TestDungeon/Sky and Fog Settings Profile.asset

```diff
@@ -27,7 +27,7 @@ MonoBehaviour:
     m_Value: 0
   multiplier:
     m_OverrideState: 1
-    m_Value: 0
+    m_Value: 0.0005
```

Comment:
This is just some ambient global light to be able to see the areas that are pitch black. For dev purposes only, until I put in some proper lighting in the rooms